backend/server/repositories/userRepoMySql.js

```diff
@@ -59,14 +81,33 @@ const createUser = async (email, password) => {
 
 const getUserInfoByEmail = async (email) => {
   var _userId = Number();
-  await findByEmail(email).then(id => _userId = id);
+  await findByEmail(email).then((id) => (_userId = id));
 
-  if (!_userId)
-    throw "No user is using " + email; 
+  if (!_userId) throw 'No user is using ' + email;
 
   return getUserInfoById(_userId);
 };
 
+const getAllUserInfo = async () => {
+  var _userInfo = [];
+
+  const selectStmt = `
+    SELECT *
+    FROM users;`;
+
+  const query = conn
+    .promise()
+    .query(selectStmt)
+    .then(([rows, fields]) => {
+      _userInfo = rows.map(({ password, ...rest }) => rest);
+    })
+    .catch(console.error);
+
+  await query;
+
+  return _userInfo;
+};
+
```

Comment:
Just a comment here that it might be possible for this to use `getUserInfo`, where if no `userId` is provided then all users info are returned.

Reply:
Will update in the next PR if there is enough time